gtk-rust-modules/src/runner.rs

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+mod module_common;
+use module_common::*;
+
+const SCARE_DIST: i32 = 10;
```

Comment:
Nit: I'd prefer this to live in the 'common' file; the idea was to put all the "game" parameters in a single place.

Reply:
I put SCARE_DIST in this because I saw it as a runner param rather than a game param (and it shows up as unused in the hunter build otherwise). I'll take this nit as optional unless otherwise specified :)

Reply:
I would argue that any params governing the behaviour of the actors form part of the game params (and also, the same constant exists in the C header).

... but it's only a demo so I don't care that much.